ui/components/app/wallet-overview/coin-buttons.tsx

```diff
@@ -323,7 +357,8 @@ const CoinButtons = ({
         event: MetaMetricsEventName.NavSwapButtonClicked,
         category: MetaMetricsEventCategory.Swaps,
         properties: {
-          token_symbol: 'ETH',
+          account_type: account.type,
```

Comment:
I think we can also include the `snap_id` here, in case it's a Snap account.

Reply:
We could, but I wonder if I should remove the `account_type` entirely here 🤔 I did not update the schema for this specific event actually...

Also, we don't support swaps for Bitcoin at the moment, so I'll revert for now!